docs/docs/behaviors/layers.md

```diff
@@ -99,7 +99,7 @@ Example:
 
 ## Toggle Layer
 
-The "toggle layer" behavior enables a layer until the layer is manually disabled.
+The "toggle layer" behavior enables a layer if it is currently disabled, or disables it if enabled.
```

Comment:
Do you know what happens if you `&tog` off a layer that is active due to an `&mo`? This reads to me as though it would disable the layer, while the original reads as if it would do nothing (though it is much less clear in that version)

Reply:
I am 90% sure that it will deactivate the momentary layer, since they all use the same on/off functions. But it might be nice to double check, I'll report later.

Reply:
I confirmed that tapping `&tog X` turns off `&mo X` even while the latter is held, so that's consistent with my understanding.